### PR TITLE
Update poster

### PR DESCRIPTION
**Pull request by onethirdzero**
@luizperes

### CODE REVIEW

File: poster/poster_file.tex [hunk 1]
```diff
@@ -146,10 +156,14 @@ \section*{Improving GrammarGuru suggestions}
 \begin{center}
 \includegraphics[width=0.8\linewidth]{proj_img}
```
Comment (onethirdzero) [inline poster/poster_file.tex:157]: Was there supposed to be an image here? I didn't see it in the final PDF.

Comment (luizperes) [reply poster/poster_file.tex:157]: It’s here https://github.com/luizperes/training-grammar-guru/blob/cmpt419/poster/proj_img.png

**Comment by luizperes** [comment]
Will review the file soon! Thank you!

**Comment by luizperes** [comment]
It looks very good! merging it!

**Pull request merged**